libs/application/templates/party-application/src/fields/EndorsementList/index.tsx

```diff
@@ -98,93 +99,105 @@ const EndorsementList: FC<FieldBaseProps> = ({ application }) => {
 
   return (
     <Box marginBottom={8}>
-      <CopyLink
-        linkUrl={window.location.href}
-        buttonTitle={formatMessage(m.endorsementList.copyLinkButton)}
-      />
-      <Box marginTop={4} display="flex" alignItems="baseline">
-        <Text variant="h2">
-          {endorsementsHook && endorsementsHook.length > 0
-            ? endorsementsHook.length
-            : 0}
-        </Text>
-        <Box marginLeft={1}>
-          <Text variant="default">
-            {formatMessage(m.endorsementList.namesCount)}
-          </Text>
-        </Box>
-      </Box>
-      <Text variant="small" color="dark300">
-        {'Leyfilegur fjöldi meðmæla í ' +
-          constituency.region_name +
-          ' er á bilinu ' +
-          minEndorsements +
-          '-' +
-          maxEndorsements}
-      </Text>
-      <Box marginTop={4}>
-        <Box
-          display="flex"
-          justifyContent="spaceBetween"
-          alignItems="center"
-          marginBottom={3}
-        >
-          <Checkbox
-            label={formatMessage(m.endorsementList.invalidSignatures)}
-            checked={showOnlyInvalidated}
-            onChange={() => {
-              setShowOnlyInvalidated(!showOnlyInvalidated)
-            }}
-          />
-
-          <Input
-            name="searchbar"
-            placeholder={formatMessage(m.endorsementList.searchbar)}
-            icon="search"
-            backgroundColor="blue"
-            size="sm"
-            value={searchTerm}
-            onChange={(e) => setSearchTerm(e.target.value)}
-          />
-        </Box>
-        <Box marginY={3}>
-          <EndorsementTable
-            application={application}
-            endorsements={endorsements}
+      {error ? (
```

Comment:
this is a large component, would it make sense to split into two separate (private - in the same file) components?